ballerina/tests/test.bal

```diff
@@ -0,0 +1,129 @@
+final string playlistsId = "3cEYpjA9oz9GiPac4AsH4n";
+final string showsId = "38bS44xjbVVZ3No3ByF1dJ";
+final string tracksId = "11dFghVXANMlKmJXsNCbNl";
+final string userId = "smedjan";
+
+// Initialize the client with auth configuration
+ConnectionConfig config = {
+    auth: {
+        token: token
+    }
+};
+
+final Client spotify = check new(config);
+
+@test:Config {}
+isolated function getAlbumTest() returns error? {
+    AlbumObject response = check spotify->/albums/[albumId];
+    test:assertTrue(response is AlbumObject);
+}
+
+@test:Config {}
+isolated function getArtistTest() returns error? {
+    ArtistObject response = check spotify->/artists/[artistId];
+    test:assertTrue(response is ArtistObject);
+}
+
+@test:Config {}
+isolated function getAudiobooksTest() returns error? {
+    AudiobookObject response = check spotify->/audiobooks/ [audiobooksId];
+    test:assertTrue(response is AudiobookObject);
+}
+
+@test:Config {}
+isolated function browseCategoriesTest() returns error? {
+    PagedCategoriesObject response = check spotify->/browse/categories;
+    test:assertTrue(response is PagedCategoriesObject);
+}
+
+@test:Config {}
+isolated function getChaptersTest() returns error? {
+    AudiobookObject response = check spotify->/chapters/[chaptersId];
+    test:assertTrue(response is AudiobookObject);
+}
+
+@test:Config {}
+isolated function getEpisodeTest() returns error? {
+    EpisodeObject response = check spotify->/episodes/[episodeId];
+    test:assertTrue(response is EpisodeObject);
+}
+
+@test:Config {}
+isolated function getMarketsTest() returns error? {
+    MarketsResponse response = check spotify->/markets;
+    test:assertTrue(response is MarketsResponse);
+}
+
+@test:Config {}
+isolated function getplaylistTest() returns error? {
+    CurrentlyPlayingContextObject response = check spotify->/playlists/[playlistsId];
+    test:assertTrue(response is CurrentlyPlayingContextObject);
+}
+
+@test:Config {}
+isolated function getShowsTest() returns error? {
+    CurrentlyPlayingContextObject response = check spotify->/shows/[showsId];
+    test:assertTrue(response is CurrentlyPlayingContextObject);
+}
+
+@test:Config {}
+isolated function getTrackTest() returns error? {
+    TrackObject response = check spotify->/tracks/[tracksId];
+    test:assertTrue(response is TrackObject);
+}
+
+@test:Config {}
+isolated function getUserTest() returns error? {
+    PrivateUserObject response = check spotify->/tracks/[tracksId];
+    test:assertTrue(response is PrivateUserObject);
+}
+
+@test:Config {}
+isolated function getAlbumTracksTest() returns error? {
+    PagingSimplifiedTrackObject response = check spotify->/albums/[albumId ]/tracks;
+    test:assertTrue(response is PagingSimplifiedTrackObject);
+}
+
+@test:Config {}
+isolated function getArtistIdTest() returns error? {
+    ArtistObject response = check spotify->/artists/[artistId];
+    test:assertTrue(response is ArtistObject);
```

Comment:
This assertion doesnt make sense because it will always evaluate to true. What we need to do is to assert a particular field in the response record (e.g. id) to see if it present and then we can assume that the response is a success response.

Reply:
@Sachindu-Nethmin FYA

Reply:
@Sachindu-Nethmin Agree with @NipunaRanasinghe, This assertion does nothing. We could at least remove the check expression.

Reply:
Shall we assert a field value where possible? using `assertEquals`. @Sachindu-Nethmin